.claude/skills/release-script/SKILL.md

```
@@ -0,0 +1,321 @@
+    "bgmVolume": 0.1,
+    "audioVolume": 1,
+    "suppressSpeech": false
+  }
+}
+```
+
+### Step 6: JSON の組み立てと保存
+
+テンプレートパラメータ + `title` + `description` + `lang` + `beats` を組み立て、以下に保存:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)